src/controllers/usersController.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-console */
+const userService = require('../services/usersServices');
+
+const get = async (req, res) => {
+  const users = await userService
+    .getAll()
+    .then((usrs) => usrs.map(userService.normalize));
```

Comment:
Inconsistent promise handling: Consider using `async/await` instead of mixing it with `.then()`. This will make the code more readable and maintainable.